test/VT100Tests.cs

```diff
@@ -140,8 +141,8 @@ public class PSMarkdownOptionInfoTests
     {
         private void ValidateDarkTheme(PSMarkdownOptionInfo opt)
         {
-            bool expectedEnableVT100 = System.Environment.OSVersion.Platform != PlatformID.Win32NT
-                || (System.Environment.OSVersion.Platform == PlatformID.Win32NT
+            bool expectedEnableVT100 = !RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+                || (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
```

Comment:
More a style comment, but I think the layout here is misleading. Initially I read this as:

```csharp
!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || Runtime.Information.IsOSPlatform(OSPlatform.Windows)
...
```

and thought to myself "but that's always true...".

Then I noticed the `&&` below and reminded myself of the binding rules for `&&` and `||` in C#.

One thing though is that if we've moved to the RHS of the `||` then `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` must have been true, so testing for it a second time is redundant.

Better to give the condition as:

```csharp
!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || Environment.OSVersion.Version.Major >= 10
```

Reply:
Fixed